backends/log_analytics/sql/5_02_users_who_accessed_most_amount_of_data.sql

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+SELECT
+  proto_payload.audit_log.authentication_info.principal_email,
+  FORMAT('%9.2f',SUM(CAST(JSON_VALUE(proto_payload.audit_log.metadata,
+      "$.jobChange.job.jobStats.queryStats.totalBilledBytes") AS INT64))/POWER(2, 40)) AS Billed_TB
+FROM
+  `[MY_PROJECT_ID].[MY_DATASET_ID]._AllLogs`
+WHERE
+  timestamp >= TIMESTAMP_SUB(CURRENT_TIMESTAMP(), INTERVAL 7 DAY)
+  AND JSON_VALUE(proto_payload.audit_log.metadata, "$.jobChange.job.jobConfig.type") = "QUERY"
```

Comment:
Minor: Since metadata is JSON, using JSONPath string is optional, so could use this more readable version:
`JSON_VALUE(proto_payload.audit_log.metadata.jobChange.job.jobConfig.type)` 

This is in line with Pattern 1 in Query migration patterns from this [blog post](https://cloud.google.com/blog/products/data-analytics/moving-to-log-analytics-for-bigquery-export-users).
Either format works, but we could choose/promote one way for consistency as more users adopt Log Analytics SQL.